holmes.go

```diff
@@ -347,6 +347,13 @@ func (h *Holmes) goroutineCheckAndDump(gNum int) {
 		return
 	}
 
+	// optimize: https://github.com/mosn/holmes/issues/84
+	// Thread dump information contains goroutine information
+	// skip goroutine dump
+	if h.threadCoolDownTime.After(time.Now()) {
```

Comment:
it's not good enough. better check it in the same check of the loop.
We'd better make sure there is a goroutine dump at the same time while generating the thread dump. otherwise, it may not friendly for later analysis.

Reply:
In the same loop?
![image](https://user-images.githubusercontent.com/39624029/174706067-fa6004cb-71ad-4317-bd35-d879f39b9587.png)

Reply:
Before checking the goroutine in the loop to determine if the thread dump was triggered?

Reply:
@doujiang24 It means that the goroutine will continue to be checked in the loop that triggers the thread dump, but the next time the goroutine is checked, it will see if it has been thread dumped?

Reply:
I come out another way:
change the `grCoolDownTime` in `threadCheckAndDump` when the thread profile is triggered.
so that we also respect the `grOpts.CoolDown`.

Reply:
just need to change gc.CoolDown at the same time as the thread dump.
ok?

Reply:
我将重新提一个PR

Reply:
yes, just using this PR is good enough.